### PR TITLE
docs: consistent formatting

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Made the formatting more consistent with the other components. Also, I think down-sampling is a more common term than sub-sampling.

Also, copied some of the text you used for the d3fc website :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-A D3 sampler that subsamples data through largest triangle and mode-median bucket methods. Data is typically partitioned into equally-sized buckets, and one data point from each bucket is chosen. The algorithms employed here are detailed in the thesis [Downsampling Time Series for Visual Representation](http://skemman.is/stream/get/1946/15343/37285/3/SS_MSthesis.pdf).
+A D3 component for down-sampling data using a variety of methods. Data is typically partitioned into equally-sized buckets, and one data point from each bucket is chosen. The algorithms employed here are Largest Triangle 1 (or 3) Bucket, or Mode-Median as detailed in the thesis [Downsampling Time Series for Visual Representation](http://skemman.is/stream/get/1946/15343/37285/3/SS_MSthesis.pdf).
 
 
 ![d3fc sample](d3fc-sample.png)
 
 For a live demo, see the [GitHub Pages site](http://d3fc.github.io/d3fc-sample/).
-
+sub
```
Comment (WPFerg) [inline README.md]: I'm assuming this wasn't intentional?

File: README.md [hunk 2]
```diff
@@ -19,58 +19,53 @@ npm install d3fc-sample
 
 ## General API
 
-The sample component provides an API to subsample data based off of their values, typically to improve performance on high-data charts. The data is passed to the component, which then returns their subsampled counterpart.
+The sampling components provide an API for downsampling data. Thye are typically used to improve rendering performance of charts or maps when there is a significant amount of data. The data is passed to the component, which then returns a smaller downsampled array.
```
Comment (WPFerg) [inline README.md]: `Thye`

File: README.md [hunk 3]
```diff
@@ -19,58 +19,53 @@ npm install d3fc-sample
...
 
----
-
-*modeMedian*.**value**(*accessor*)
+*modeMedian*.**value**(*accessorFunc*)
 
-Specifies the 'value' for each item in the associated array.
-
----
+Specified the accessor function used to obtain the value from the supplied array of data. The accessor function is invoked exactly once per datum, and should return the value to be down-sampled.
```
Comment (WPFerg) [inline README.md]: Should this be `Specifies`?

File: README.md [hunk 4]
```diff
@@ -78,42 +73,32 @@ The largest triangle algorithm comes in two flavours -- one bucket and three buc
...
 *largestTriangle*.**bucketSize**(*size*)
 
 Denotes the amount of data points for each bucket. The first and last data points are always their own bucket. The second-last bucket will be of size `(data.length - 2) % size`.
 
----
-
 **largestTriangle**(*data*)
 
-Runs the subsampler. It returns the subsampled data (it doesn't modify the `data` array itself). The subsampler selects the point in the bucket with the largest area between two other points (determined by algorithm).
+Runs the ssampler, returning the downsampled data (it doesn't modify the `data` array itself). The sampler selects the point in the bucket with the largest area between two other points (determined by algorithm).
```
Comment (WPFerg) [inline README.md]: `ssampler`

File: README.md [hunk 5]
```diff
@@ -78,42 +73,32 @@ The largest triangle algorithm comes in two flavours -- one bucket and three buc
...
 
-*largestTriangle*.**x**(*accessor*)
+*largestTriangle*.**y**(*accessorFunc*)
 
-*largestTriangle*.**y**(*accessor*)
-
-Specifies the x/y value for each item in the associated array, used to calculate the area between points.
-
----
+Specified the accessor function used to obtain the x and y values from the supplied array of data. The accessor function is invoked exactly once per datum, and should return the value to be down-sampled.
```
Comment (WPFerg) [inline README.md]: `Specifies` again?

**Comment by WPFerg** [comment]
Thanks -- I've spotted a few typos but otherwise looks good :)

**Comment by ColinEberhardt** [comment]
@WPFerg - that'll teach me to edit a markdown file without a spell-checker! I've fixed those issue.

**Pull request merged**